XamarinCommunityToolkit/Converters/StateToBooleanConverter.shared.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+using Xamarin.CommunityToolkit.UI.Views;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class StateToBooleanConverter : IValueConverter
```

Comment:
Should it be inherited from ValueConverterExtension as well as other converters?
+ Add a StateToCompare property